ai_presenter/voice_ai/base.py

```diff
@@ -3,6 +3,14 @@
 import logging
 
 
+class VoiceAIActor:
+    def __init__(self, config: VoiceConfig):
+        self.config = config
```

Comment:
This is good, but instead of having it point to a config, copy the data to `self.name`, etc. That way the object will have clear information of the data it contains